src/ui/view-subscribe.ts

```diff
@@ -0,0 +1,36 @@
+import { property, html, customElement, css } from 'lit-element'
+import { Connected, State, MyModelSelectors } from './connected'
+import { sharedStyles } from './shared-styles'
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'view-subscribe': ViewSubscribeElement
+  }
+}
+
+@customElement('view-subscribe')
+export class ViewSubscribeElement extends Connected {
+  @property({ type: String })
+  myprop: ""
+
+  mapState(state: State) {
+    return {
+      myprop: MyModelSelectors.squaredValue(state),
+    }
+  }
+
+  render() {
+    return html `<p>From the dispatch Web Component: ${this.myprop}</p>`
+  }
```

Comment:
Instead of a separate element to show the result (also on a separate route), it would make sense to combine it all into one element.

This makes it all look more complicated than is necessary, especially as the naming of the routes / elements implies some part in things.